README.md

```diff
@@ -159,15 +159,18 @@ The settings file is a JSON file where all configurable elements of the project
 The following table containers all of the the current supported fields in the settings file.
 
 _Settings_
-|variable|type|description|
-|--------|----|-----------|
-|log_level|string|sets the log level in application logs (debug, info, warn, error, fatal)|
-|stub_resource|string|enables or disables the use of stub resources for unrecognized events {"enabled" or "disabled"}|
-|name|string|sets the "application_name" field set in all log entried|
-|timestamp_format|string|golang timestamp format used when a human readable timestamps are provided|
-|auditing.required_tags|string array| a list of labels/tags that will be audited for by the tag auditor|
-|actions.mode|string|selector for whether or not Cloud Inquisitor will take actions against a resource or just log if it would have|
-|newrelic.license|string|New Relic license key if taking advantage of New Relic logging|
+|variable|type|values|description|
```

Comment:
Please wrap discrete values in back ticks.  That way the any fields stand out as taking any value and not the value `any`